templates/web/base/alert/index.html

```diff
@@ -37,16 +37,16 @@
 </form>
 
 [% IF photos.size %]
-<div id="alert_recent">
-  <aside>
-    <h2>[% loc('Some photos of recent reports') %]</h2>
-    [% FOREACH p IN photos;
-        photo = p.get_photo_params;
-    %]
-        <a href="/report/[% p.id %]"><img border="0" height="100"
-            src="[% photo.url_tn %]" alt="[% p.title | html %]" title="[% p.title | html %]"></a>
-    [% END %]
-  </aside>
+<h2>[% loc('Some photos of recent reports') %]</h2>
```

Comment:
The styling for `.alerts__nearby-activity` just adds a grey background, so I left it out on the alert index page because there's already an element immediately above with a similar grey background (the main postcode search form).